### PR TITLE
#681: Footer Covers Drop-Down Options in Consent Wizard Step 2

### PR DESCRIPTION
**Pull request by shailikadakia-oicr**
### Consent UI
Added the z-index for `footer` so that the content above has a greater priority. This way the dropdown options lay on top of the dropdown instead of under. 
## PR Readiness Checklist

- [x] "Expected Outcome(s)" in ticket have been met
- [x] Ticket number included in PR title
- [x] Connected ticket to PR
- [x] Labels added to PR for service name (`consent-api`, `data-mapper`, etc...), type (`chore`, `documentation`, etc...), status (`draft`, `on-hold`, etc...) **if applicable**
- [x] Manual testing completed
- [x] Builds locally without errors or warnings
- [ ] Tests are updated (if required) and passing
- [ ] [Axe DevTools](https://www.deque.com/axe/browser-extensions/) "Full Page Scan" of updated pages passes without introducing new Critical or Serious issues (UI changes only)
- [ ] PR feedback has been addressed **if applicable**
- [ ] New environment variables added to `.env.schema` files, `README.md`
- [ ] Added copyrights to new files

### CODE REVIEW

**Review by anncatton — COMMENTED** [review]
The dropdowns look good on Step 2 now! But looks like you'll just need to tinker with the  layout on the `/register` and `/invite` pages, as the sidebar image now overlaps the footer:
image

**Comment by shailikadakia-oicr** [comment]
Thank you @anncatton I've made some changes to fix the bug :)

File: apps/consent-ui/src/components/common/Form/fieldsets/SelectFieldSet/SelectFieldSet.module.scss [hunk 1]
```diff
@@ -64,6 +64,7 @@
 				&-list {
 					padding: 0;
 				}
+				z-index: 2;
```
Comment (mistryrn) [inline apps/consent-ui/src/components/common/Form/fieldsets/SelectFieldSet/SelectFieldSet.module.scss:67]: Super smol change request, do you mind moving this just after line 63 instead, so it's with the rest of the styles for this selector?

File: apps/consent-ui/src/components/common/Footer/Footer.module.scss [hunk 2]
```diff
@@ -14,7 +14,9 @@
 	@media (min-width: theme('screens.xl')) {
 		@apply grid-cols-5 grid-rows-2;
 	}
-	z-index: 5;
+	z-index: 1;
+	position: relative;
+	overflow: hidden;
```
Comment (mistryrn) [inline apps/consent-ui/src/components/common/Footer/Footer.module.scss:19]: Similarly to the other comment, I know this `z-index` declaration was here before you started working on this but let's move these lines to just after line 8 to be with their friends 🙂

**Review by mistryrn — COMMENTED** [review]
Tested locally, works great! Just a couple _very minor_ change requests and then this should be good to go

**Review by mistryrn — APPROVED** [review]
🚀

**Pull request merged**